modules/FP_events/metadata/subpaneldefs.php

```diff
@@ -54,7 +54,7 @@
 			'module'=>'Delegates',
 
 			'top_buttons' => array(
-				array('widget_class' => 'SubPanelTopButton_c'),
+				//array('widget_class' => 'SubPanelTopButton_c'),
```

Comment:
Dear Ashley

Thanks for your comment, i will try next time to remove all old or commented code.

Jerzy
________________________________
From: Ashley Nicolson <notifications@github.com>
Sent: 13 July 2016 15:21:33
To: salesagility/SuiteCRM
Cc: Jerzy Brzezinski; Comment
Subject: Re: [salesagility/SuiteCRM] Adding Events Delegates Filter via Subpanel (#1709)


In modules/FP_events/metadata/subpaneldefs.php<https://github.com/salesagility/SuiteCRM/pull/1709#discussion_r70633993>:

> @@ -54,7 +54,7 @@
>                       'module'=>'Delegates',
>
>                       'top_buttons' => array(
> -                             array('widget_class' => 'SubPanelTopButton_c'),
> +                             //array('widget_class' => 'SubPanelTopButton_c'),


Remove rather than comment

-
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/salesagility/SuiteCRM/pull/1709/files/daa22c19c56a4c1de1cf1a62a308172c75944a2c..35bb4c9423487593884cfa650091a2b8898c0711#r70633993>, or mute the thread<https://github.com/notifications/unsubscribe/AKP8DHSaYC9oQ8THVDeoLnOj6xRL3KfMks5qVPRtgaJpZM4JF9xd>.